src/layouts/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
```

Comment:
ops. So -- since you came late you might have missed this context -- we want this to be a static site, this is why we are using a static site generator such as gatsby. We do not want react or any other JS framework in place. Can you please refactor this so that these are either html files or MD files? Thanks!

Reply:
This is a static site. React is only used as a syntax for declaring and composing components.